src/main/kotlin/venus/Renderer.kt

```diff
@@ -95,16 +95,20 @@ internal object Renderer {
 
     fun renderSimButtons() {
         val simbtns = document.getElementById("simulator-buttons") as HTMLDivElement
-        val simassmbbtns = document.getElementById("simulator-assemble-buttons") as HTMLDivElement
-        simassmbbtns.style.display = "none"
+        // Turn all simulator-related buttons on

```

Comment:
can this comment be moved up/down one line and same with the comment on line 107

Reply:
Done